metrics/nodehealthcheck.go

```diff
@@ -32,15 +32,42 @@ var (
 	)
 )
 
+var (
+	// NodeHealthCheckUnhealthyNodeOngoingRemediation is a Prometheus metric, which reports an ongoing remediation of a node
+	NodeHealthCheckUnhealthyNodeOngoingRemediation = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "nodehealthcheck_unhealthy_node_ongoing_remediation",
```

Comment:
Can we have a shorter name please? Remediations are always for unhealthhy nodes, so IMHO this information is redundant 🙂 What about `nodehealthcheck_ongoing_remediations`?

Reply:
Sure, good call